src/test/java/domain/LottoGeneratorTest.java

```diff
@@ -0,0 +1,16 @@
+package domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LottoGeneratorTest {
+    @Test
+    @DisplayName("로또생성기 생성")
+    void createLotto() {
+        // when
+        LottoDto lottoDto = LottoGenerator.generate();
+        // then
+        Assertions.assertThat(lottoDto.getLottoNumbers().size()).isEqualTo(6);
+    }
```

Comment:
Assertions 는 static import를 해주시면 더 깔끔해집니다~
다른 코드들도 static import로 바꿔주세요!

```suggestion
        assertThat(lottoDto.getLottoNumbers().size()).isEqualTo(6);
```

Reply:
바로 수정하겠습니다.